.ruby-version

```diff
@@ -1 +1 @@
-2.7.5
+3.0.3
```

Comment:
might as well go to 3.0.4, since that has been released since we first attempted the upgrade: 

https://www.ruby-lang.org/en/news/2022/04/12/ruby-3-0-4-released/

Reply:
done, bumped it in capi-release pr